src/services/emailService.js

```diff
@@ -2,20 +2,35 @@ import formData from 'form-data';
 import Mailgun from 'mailgun.js';
 import config from '../config.js';
 import validator from 'validator';
+import Logger from '../tools/logger.js';
 
-const { DOMAIN } = config;
-const mailgun = new Mailgun(formData);
-const client = mailgun.client({
-  username: config.EMAIL_USERNAME,
-  key: config.EMAIL_KEY,
-});
+let EMAIL_CLIENT;
```

Comment:
In a file like this, does the code that's not included in a function just run whenever emailService gets called? Basically do lines 8-15 get run everytime this emailService is used?